pandas/tests/frame/methods/test_replace.py

```diff
@@ -661,6 +661,14 @@ def test_replace_simple_nested_dict_with_nonexistent_value(self):
         result = df.replace({"col": {-1: "-", 1: "a", 4: "b"}})
         tm.assert_frame_equal(expected, result)
 
+    @pytest.mark.parametrize("to_replace, value", [(np.nan, pd.NA), (np.nan, None)])
+    def test_replace_numpy_nan(self, to_replace, value):
+        # GH#45725 ensure numpy.nan can be replaced with pandas.NA or None
+        df = DataFrame({"A": [to_replace]}, dtype=object)
+        result = df.replace({to_replace: value})
```

Comment:
can you also test usage with `df.replace(to_replace, value)` or is this tested close by here?

Reply:
Yes done.